src/data/list/basic.lean

```diff
@@ -1327,15 +1327,11 @@ lemma last_eq_nth_le : ∀ (l : list α) (h : l ≠ []),
 | (a :: b :: l) h := by { rw [last_cons, last_eq_nth_le (b :: l)],
                           refl, exact cons_ne_nil b l }
 
-lemma nth_le_length_sub_one {l : list α} (h₁ h₂) :
```

Comment:
Why did you remove this lemma?

Reply:
Because it was almost the same as `last_eq_nth_le`. Do you want to have both?

Reply:
It's worth having both as long as you derive the hypotheses in the opposite direction as I mentioned above.

Reply:
Oh! Done. Do you want to golf the proof?